src/stores/lock-store-base.js

```diff
@@ -3,20 +3,23 @@ import StoreBase from './store-base.js';
 import uniqueId from '../utils/unique-id.js';
 
 class LockStoreBase extends StoreBase {
```

Comment:
As this class is now base class to only one remaining lock store, I think we do not need it anymore, we could just pull up all members into the actual `LockStore`.

Reply:
I was thinking if I should do that or keep the separation 🤔 Ok, will merge it into the `LockStore`.

Reply:
Merged, thanks!